README.md

```diff
@@ -1,13 +1,46 @@
-# RomCom
+#RomCom Group Project (Turing School of Software and Design)
+###(Turing School of Software and Design)
 
-A boilerplate repo. 
+## Table of Contents
+  - [Abstract](#abstract)
+  - [Technologies](#technologies)
+  - [Illustrations](#illustrations)
+  - [Install + Setup](#set-up)
+  - [Contributors](#contributors)
+	- [Wins](#wins)
+	- [Challenges + Improvements](#challenges-+-Improvements)
+  - [Project Specs](#project-specs)
 
-## Set Up
-1. ONE teammate: fork this boilerplate repository 
-2. Clone down your new, forked repo
-3. cd into the repository
-4. Open it in your text editor
-5. View the project in the browser by running open index.html in your terminal
-6. Add all project partners and your assigned instructor as collaborators on the repository
+## Abstract
+	- Utilize Javascript to manipulate the HTML elements on a webpage for displaying/
```

Comment:
Our text is showing up as a code block (inside a grey box) and our bullet points are not showing up.